configuration/blockly/rules-blockly-persistence.md

```diff
@@ -84,20 +93,44 @@ Notes: in case no or 0 values are retrieved, make sure that the item in question
 
 ![previous-example](../images/blockly/blockly-persistence-get-previous-example.png)
 
-### Check item change / update since a point in time
+### Check item change / update
 
 ![item-change](../images/blockly/blockly-persistence-get-item-change.png)
 
-_Function:_ checks if an item was updated or changed since a certain point in time
+_Function:_ checks if an item was updated or changed since the time provided by _ZonedDateTime_-Block, will be updated or changed until the time provided by _ZonedDateTime_-Block, or between 2 timepoints provided by _ZonedDateTime_-Blocks.
+
 Type: boolean `true` or `false`
 
-### Provide last updated date of an Item
+### Provide last or next updated or changed date of an Item
 
 ![item-updated-date](../images/blockly/blockly-persistence-updated-date.png)
 
-_Function:_ Provides the last updated date (including time) of an Item
+_Function:_ Provides the last or next updated or changed date (including time) of an Item
 Type: ZonedDateTime
 
+### Persist State or States for an Item
+
+Persist State or States for an Item.
+
+Multiple options are available:
+
+- state (at current time): persist given State at the current time
+
+Following options are not supported by all persistence services (for example the standard rrd4j does _not_ support this).
+
+- state (at specific time): persist given State at given time
+- list of states (adding): persist list of States, not replacing the already persisted States
```

Comment:
Do you mind giving a example where this needed or helpful of if explained somewhere else link to it? I don't even know what that is for (sorry 😬)

Reply:
It looks like documentation to link to is meager at best. This has been introduced in the context of the forecast persistence strategy, where it became useful to persist/retrieve a series of persisted values, not just one. I will try to formulate something to make it clearer.